zetaclient/bitcoin/bitcoin_client_rpc_test.go

```diff
@@ -43,8 +45,8 @@ func (suite *BitcoinClientTestSuite) SetupTest() {
 	tss := interfaces.TestSigner{
 		PrivKey: privateKey,
 	}
-	//client, err := NewBitcoinClient(common.BtcTestNetChain(), nil, tss, "", nil)
-	client, err := NewBitcoinClient(common.BtcRegtestChain(), nil, tss, "/tmp", nil, log.Logger, config.BTCConfig{}, nil)
+	appContext := appcontext.NewAppContext(&corecontext.ZeraCoreContext{}, &config.Config{}, log.Logger)
```

Comment:
We have the `testutil/sample` package to define test purpose placeholder values, maybe we can add it here?
NOTE: if it create a circular dependency it means we should use `bitcoin_test` for this package which might a change out of scope for this PR